### ISSUE TITLE
Cannot use break in a while loop

### ISSUE THREAD

**Issue by xuan-li** [NVIDIA/warp#196]
To reproduce:

```python
import warp as wp

@wp.func
def test_func():
    stop = wp.bool(False)
    while not stop:
        stop = True
        break

@wp.kernel
def test_kernel():
    test_func()

wp.init()
wp.launch(test_kernel, dim=1)
```

Got the following exception:
```
Exception: CUDA kernel build failed with error code 6
```

removing ```break``` works fine.

**Comment by c0d1f1ed** [comment]
Thanks for reporting this! A fix will be available in the next release.

**Comment by KinjalParikh** [comment]
Similarly `continue` keyword doesn't work either

**Comment by cadop** [comment]
I think both break and continue where raised as issues before as well https://github.com/NVIDIA/warp/commit/b57868fac70cce0e585ee0053f038fc3c9e25356

**Comment by c0d1f1ed** [comment]
This is fixed as part of [Warp 1.1.0](https://github.com/NVIDIA/warp/releases/tag/v1.1.0)!
